.gitignore

```diff
@@ -102,3 +102,6 @@ gen
 save_test1.txt
 save_test2.txt
 save_test3.txt
+save_test1.json
+save_test2.json
+save_test3.json
```

Comment:
don't need them if you're removing them at the end of tests, right?

Reply:
In some cases, the `os.remove` function fails to delete files due to a lack of privileges, so I added all of them to `.gitignore`.